packages/core/src/internal/proc.js

```diff
@@ -40,21 +40,19 @@ function getIteratorMetaInfo(iterator, fn) {
   return getMetaInfo(fn)
 }
 
-// TODO: check if this hacky toString stuff is needed
-// also check again whats the difference between CHANNEL_END and CHANNEL_END_TYPE
+// toString is used to have ability to log actions in devtools.
```

Comment:
do we really need it? browser devtools can show Symbol correctly. any custom solution could achieve it with `String(symbol)`

Reply:
I think this comment was referring to https://github.com/redux-saga/redux-saga-devtools but this project is unmaintained atm, so probably it doesnt matter much and as mentioned other workarounds exist

Reply:
should we delete the construction in this case?

Reply:
which one? you already have deleted `CHANNEL_END` from here, which I think is fine 👍 

I think this comment "toString is used to have ability to log actions in devtools." might be removed too.